json_schemas/users.json

```diff
@@ -22,6 +22,11 @@
       "type": "string",
       "pattern": "^(buyer|supplier|admin)$"
     },
+    "supplier_id": {
+      "type": "string",
```

Comment:
I know what you mean, but if DB is numeric already then maybe we just stick with that? Service IDs changed to retrofit G4 and G5 services, but I can't immediately think of a scenario where we'd want to change supplier ID to string in future?  We will need to come up with some way of generating supplier IDs before long though - at the moment they are just the old database IDs.